### PR TITLE
Create FAQ.md

### PR DESCRIPTION
**Pull request by PumPumDev**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Frequent Questions
+
+## Q: What docker version should I use if I have Windows 10 Home?
+A: Use [Docker Toolbox] (https://docs.docker.com/toolbox/toolbox_install_windows/)
+
+## Q: How does is works?
+A: You just have to download the latest .exe.After the instalation, when you turn it on it runs a Virtual Machine which use "normal" docker.
```
Comment (jserranohidalgo) [inline FAQ.md]: Minor typos: 

- "... the latest `.exe`. After ... ", quote `exe`, and add whitespace. 
- "which uses"

**Review by jserranohidalgo — CHANGES_REQUESTED** [review]
Great start!
Thanks!

**Comment by PumPumDev** [comment]
I've just changed the file

**Comment by jserranohidalgo** [comment]
@alfonsorr @nmartinezs2017 @correosoria @yh89 Any additional question?

**Pull request merged**